CHANGELOG.md

```diff
@@ -1,10 +1,26 @@
-# Unreleased
+# Changelog
+All notable changes to this project will be documented in this file.
 
-# v0.2.0
-* Provider is now one file, for easier distribution
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
```

Comment:
When I run the `parse-changelog.sh` script locally, it says that the `https` on these two lines should be `http`, and the comma on line 4 needs to be deleted:
```
The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
```